lib/Yam/Agama/Pom/GrubMenuSlesPage.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles grub screen in SLES.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::Agama::Pom::GrubMenuSlesPage;
+use strict;
+use warnings;
+use testapi;
+
+sub new {
+    my ($class, $args) = @_;
+    return bless {
+        grub_menu_base => $args->{grub_menu_base},
+        tag_first_entry_highlighted => 'grub2',
```

Comment:
grub-menu-sles16-highlighted

Reply:
Already created as `grub-menu-SLES-highlighted`

Reply:
Renamed to `grub-menu-sles16-highlighted`